### DISCUSSION TITLE
Converting a Docker Install into Ansible..

### DISCUSSION THREAD

**Discussion by pwrj** [AzuraCast/AzuraCast#3891] [Q&A]
I was wondering, is it possible to convert my existing install from Docker to Ansible? Or is it possible to Backup my Docker Install and restore it on an Ansible installation?

**Comment by BusterNeece** [comment]
It most likely isn't possible to migrate from Docker to Ansible, and we don't officially plan to build a way to do that, as it would be moving from a supported to an unsupported installation method.

**Reply by pwrj** [reply]
Thank you @SlvrEagle23 - Make sense. Is it therefore not recommended to install via the Ansible method?

**Reply by BusterNeece** [reply]
@pwrj Correct.
